bucket/foxit-reader.json

```diff
@@ -6,26 +6,27 @@
         "identifier": "Freeware",
         "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
     },
-    "url": "https://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/win/9.x/9.7/en_us/FoxitReader97_Setup_Prom_IS.exe#/dl.7z",
-    "hash": "341e29058ee6ba4e952980fe07e68a36579706f5116fda0e26736f11809a96d7",
+    "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/9.7/D205F785AF753E06041933E5C5E19E5C/FoxitReader97_enu_Setup_Prom.exe#/dl.7z",
+    "hash": "md5:d205f785af753e06041933e5c5e19e5c",
     "bin": "FoxitReader.exe",
     "shortcuts": [
         [
             "FoxitReader.exe",
             "Foxit Reader"
         ]
     ],
-    "installer": {
-        "script": [
-            "Get-ChildItem \"$dir\\`$PLUGINSDIR\" '*_Prom.exe' | Select-Object -ExpandProperty Fullname | Expand-InnoArchive -DestinationPath \"$dir\"",
-            "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse"
-        ]
+    "innosetup": {
+        "type": true
     },
     "checkver": {
         "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
-        "jsonpath": "$.package_info.version[0]"
+        "regex": "((\\d+\\.*)+).*(?<md5hash>([a-fA-F0-9]{32}))"
     },
     "autoupdate": {
-        "url": "https://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/win/$majorVersion.x/$majorVersion.$minorVersion/en_us/FoxitReader$majorVersion$minorVersion_Setup_Prom_IS.exe#/dl.7z"
+        "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/$majorVersion.$minorVersion/$matchMd5hash/FoxitReader$majorVersion$minorVersion_enu_Setup_Prom.exe#/dl.7z",
+        "hash": {
+            "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
+            "find": "$md5"
```

Comment:
```suggestion
            "regex": "$md5"
```

Reply:
I don't see any changes.
@linsui is right. You should user `regex` instead of `find`.